README.md

```diff
@@ -208,7 +190,7 @@ To this:
 
 ```js
 actions: {
-  multiply: (state, payload) => state * payload
+  multiply: (state, payload) => state * payload;
```

Comment:
This is a syntax error. Prettify was probably tricked because this is a partial code-listing, missing the full expression.

Reply:
I see. It can be fixed by dropping the language (`js`) extension for the partial code blocks (syntax highlighting is going to be broken for the invalid code anyway).